types/types.go

```diff
@@ -185,7 +185,7 @@ type UnparsedImage interface {
 	// (not as the image itself, or its underlying storage, claims).  This can be used e.g. to determine which public keys are trusted for this image.
 	Reference() ImageReference
 	// Close removes resources associated with an initialized UnparsedImage, if any.
-	Close()
+	Close() error
```

Comment:
The callers in `copy.Image` (both for `UnparseImage` and `Image`) should be updated to handle the error.

And perhaps `doc.go` as well?